src/geo/tiles.ts

```diff
@@ -132,3 +135,118 @@ export function tileToBbox(tile: ITile): BBox2d {
   const tileSize = degreesPerTile(tile.zoom);
   return [minPoint.longitude, minPoint.latitude, minPoint.longitude + tileSize, minPoint.latitude + tileSize];
 }
+
+/**
+ * returns the batch area
+ * @param ITileRange
+ * @returns
+ */
+export function batch2area(batch: ITileRange): number {
+  return (batch.maxX - batch.minX) * (batch.maxY - batch.minY);
+}
+
+/**
+ * returns tiles amount of given feature and zoom ranges - based on 2:1 tile scheme
+ * @param feature
+ * @param maxZoom
+ * @param minZoom optional - default as 0
+ * @returns tile count included on provided feature and zooms ranges
+ */
+export function featureToTilesCount(feature: Feature<Polygon | MultiPolygon>, maxZoom: number, minZoom = 0): number {
```

Comment:
delete this method as we don't want to expose this algorithm because of the out of memory issue